### ISSUE TITLE
Admin: lang attribute on HTML tag is always en

### ISSUE THREAD

**Issue by gu-stav** [strapi/strapi#13038]
## Bug report

Even if the language of the admin app is changed, the `lang` attribute is set to `en`, which prevents screenreaders from using the right pronunciation. It should be set according to the language selected by the user.

### Required System information

- Strapi version: 4.1.7

### Steps to reproduce the behavior

1. Change the language of the admin interface in the user profile
2. Inspect the `lang` attribute of the `html` element

### Expected behavior

The `lang` attribute should be set according to the user's language.

**Comment by fdis111** [comment]
Hey @gu-stav, I would like to work on this issue.

**Comment by Emmanuel-Melon** [comment]
> Hey @gu-stav, I would like to work on this issue.

@fdis111 any updates on this?
